hera_mc/sys_handling.py

```diff
@@ -77,7 +77,7 @@ def get_dubitable_list(self, date='now', return_full=False):
 
         raise ValueError('Too many open dubitable lists ({}).'.format(len(fnd)))
 
-    def get_all_fully_connected_at_date(self, at_date, station_types_to_check=['HH']):
+    def get_all_fully_connected_at_date(self, at_date, station_types_to_check=['HH', 'HA', 'HB']):
```

Comment:
I see three or four places in the overall `hera_mc` codebase that include a hardcoded list of station types like this. If/when we add more station types to consider, we might get bugs if not all of the lists are updated. Would it be better to make this list a global variable somewhere? (One can write, e.g., `def my_func(station_types=cm_utils.default_station_types)`, if the `cm_utils` has been imported in the module.)

Reply:
Why does this need to be `my_func?  I did a version of this.